Makefile

```diff
@@ -63,8 +61,6 @@ deps-ubuntu:
 		tesseract-ocr-eng \
 		tesseract-ocr \
 		wget
```

Comment:
We don't need `wget` anymore then, do we?
```suggestion
```

Reply:
No, plus `wget` comes with ocrd/core.

Reply:
Removed now.